recipes/filter-api-validator.rb

```diff
@@ -0,0 +1,25 @@
+#
+# Cookbook Name:: repose
+# Recipe:: filter-api-validator
+#
+# Copyright (C) 2015 Rackspace Hosting
+#
+
+include_recipe 'repose::install'
+
+unless node['repose']['filters'].include? 'api-validator'
+  filters = node['repose']['filters'] + ['api-validator']
+  node.normal['repose']['filters'] = filters
+end
+
+template "#{node['repose']['config_directory']}/validator.cfg.xml" do
+  owner node['repose']['owner']
+  group node['repose']['group']
+  mode '0644'
+  variables(
+    :wadl => node['repose']['api_validator']['wadl'],
+    :dot_output => node['repose']['api_validator']['dot_output'],
+    :enable_rax_roles => node['repose']['api_validator']['enable_rax_roles'],
+  )
+  notifies :restart, 'service[repose-valve]'
+end
```

Comment:
Probably want to add a minitest test for this template resource.  There are examples in https://github.com/rackerlabs/cookbook-repose/tree/master/files/default/test